### ISSUE TITLE
unused-variable not triggered for bound exceptions

### ISSUE THREAD

**Issue by PCManticore** [PyCQA/pylint#2480]
### Steps to reproduce
1. Save the following file and run `pylint` over it
```
try:
   1/0
except Exception as e:
   pass
```

### Current behavior
`pylint` does not complain that `e` is unused.

### Expected behavior
We should complain about that.

### pylint --version output
master

**Comment by sushobhit27** [comment]
Can close this one https://github.com/PyCQA/pylint/issues/2223

**Comment by PCManticore** [comment]
Oh thank you @sushobhit27 should have checked before!
